### ISSUE TITLE
Add /item command separately

### ISSUE THREAD

**Issue by Fiks75** [Zrips/CMI#495]
Please add /item command separately as in Essentials.
In your plugin /item command works together with the command /cmi give. It is not right.

Command /item - Gives **yourself** a specified item. (no stack item, examle totems)
Command /give - Give **another** player an item.
These commands are different, but you used them as general.

Add permissions to the command /item:
cmi.command.item | Allow access to the /item command. (give not stack items, examle totems)
cmi.command.item.oversizedstacks | Allows spawning of stacks above normal max stack size. (give stack items)

CMI plugin version: 7.8.3.0
Server version: Spigot 1.12.2-R0.1-SNAPSHOT
Vault version: 1.5.6-b49
CraftBukkit version git-Spigot-2086bb0-8cc5a7e

**Comment by Gasha666** [comment]
CMI have aliases you can turn off /item command in CMI aliases and only use /cmi item

my honest advice get rid of essentials, you don't need anything from essentials since CMI have ALL features

**Comment by CloudeLecaw** [comment]
i totally agree with TheWranger. There is no Point of adding this jus because Essentials has it.
u allready can give you or others with the give command.
and a Special items does not make sense.
i /give me/others items  me can be playername and is optional.
i u dammly hard Need this /item then go into CustomAlais.yml and add:
```
  item:
  - cmi give [playerName] $1
```

**Comment by TheWranger** [comment]
A better solution would be to add a permission to `/give` for giving items to other players.
CMI probably already has this though. I can't check right now since I'm not at my computer.
